app/views/forms/_form.json.jbuilder

```diff
@@ -1,5 +1,5 @@
 json.extract! form, :id, :name, :description, :created_at, :updated_at, :questions, \
```

Comment:
I think that @rdingwell said in Slack that we aren't putting publisher info in the API.